changes/4412.added

```diff
@@ -0,0 +1 @@
+Added `locations` field to VLAN.
```

Comment:
Also should be documented in the `VLAN` model documentation and summarized in the 2.2 release notes release overview.